src/api/burn/WixToolset.Mba.Core/IBootstrapperApplication.cs

```diff
@@ -1450,6 +1445,25 @@ public enum ErrorType
         Apply,
     };
 
+    /// <summary>
+    /// Indicates the source of the FilesInUse message.
+    /// </summary>
+    public enum FilesInUseType
+    {
+        /// <summary>
+        /// Generated from INSTALLMESSAGE_FILESINUSE.
+        /// </summary>
+        Msi,
+        /// <summary>
+        /// Generated from INSTALLMESSAGE_RMFILESINUSE.
+        /// </summary>
+        MsiRm,
```

Comment:
Should we be spelling out Restart Manager (here and/or in the native code)?

Reply:
I guess I was following Microsoft's lead here, they always used RM - `INSTALLMESSAGE_RMFILESINUSE`, `MsiRMFilesInUse` (Dialog), etc.

Reply:
Yeah, that's a typical Microsoft native-code approach. Not so much with managed code, though. And Burn hasn't shied away from long-ass names...